### PR TITLE
Better UX for CreateList Modal

### PR DESCRIPTION
**Pull request by MauroGuerrero20**
# Features
- User no longer needs to search for the same book they clicked on.
- Book is tracked and displayed in the modal.

# Demo 
UX No List Book Tracking Demo

### CODE REVIEW

File: capstone-frontend/src/components/CreateList.jsx [hunk 1]
```diff
@@ -84,6 +97,8 @@ class CreateList extends Component {
 
     // Rerender
     this.setState({ addedBooksIDs: this.state.addedBooksIDs, addedBooks: this.state.addedBooks })
+
+    console.log(this.state.addedBooksIDs, this.state.addedBooks, this.state)
```
Comment (antoniolinhart) [inline capstone-frontend/src/components/CreateList.jsx]: nit: remove console.logs

File: capstone-frontend/src/components/CreateList.jsx [hunk 2]
```diff
@@ -28,6 +28,19 @@ class CreateList extends Component {
     }
   }
 
+  initialSelectedBook = () => {
+    if (this.props.selectedBookID && this.props.selectedBook &&
+      !this.state.addedBooksIDs.includes(this.props.selectedBookID) && !this.state.addedBooks.includes(this.props.selectedBook)) {
```
Comment (antoniolinhart) [inline capstone-frontend/src/components/CreateList.jsx]: Break this up into multiple lines for each condition

**Review by antoniolinhart — APPROVED** [review]
LGTM! I think this change makes the user experience make more sense

**Review by steven-solar — APPROVED** [review]
Nice change! This definitely makes the user flow make more sense.

**Review by VivekShankar — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/CreateList.jsx (hunk 2 above)
Comment (MauroGuerrero20) [reply capstone-frontend/src/components/CreateList.jsx]: Ok

File: capstone-frontend/src/components/CreateList.jsx (hunk 1 above)
Comment (MauroGuerrero20) [reply capstone-frontend/src/components/CreateList.jsx]: Ok

**Pull request merged**